topics/imaging/tutorials/detection-of-mitoflashes/tutorial.md

```diff
@@ -0,0 +1,152 @@
+>    - **Boundary pixels** (`bndy`): Set to `10` to ignore spots within the image edges, adjusting as needed for image size.
+>    This tool detects mitochondria as spots or 'mitoflashes' based on intensity maxima, identifying them as distinct regions. Adjusting the threshold parameter can improve the accuracy of spot detection, allowing for better identification of mitoflash events.
+>
+{: .hands_on}
+
+## Step 2: Tracking Mitochondrial Movement
+
+> <hands-on-title>Tracking Mitochondrial Movement</hands-on-title>
+>
+> 1. {% tool [Perform linking in time series (nearest neighbors)](toolshed.g2.bx.psu.edu/repos/imgteam/points_association_nn/ip_points_association_nn/0.0.3-2) %} with the following recommended parameters:
+>    - {% icon param-file %} *"Coordinates (and intensities) of input points"*: Output from the **Spot Detection** tool.
+>    - **Neighborhood size** (`nbpx`): Set to `6` pixels as a starting value. Adjust within `1-10` pixels based on the density of points and proximity between spots across frames.
+>    - **Intensity threshold** (`thres`): A recommended starting value is `25%` of the global maximum intensity. Increase to `30-40%` for stricter filtering if spots have low intensity.
+>    - **Minimum track length** (`minlen`): Use `50%` of the sequence length as a baseline. Adjust based on the expected duration of mitoflash events.
+>    
+>    This tool links detected spots across frames to track movement. The **Neighborhood size** parameter controls the pixel range for associating points between frames. Increasing the **Intensity threshold** filters out weaker signals, improving track accuracy.
+>
+{: .hands_on}
+
+# Detecting Mitoflashes
+
+Mitoflashes are identified based on sudden changes in fluorescence intensity in mitochondria, signifying superoxide bursts. We will use a curve-fitting tool to capture these fluctuations and measure their characteristics.
+
+## Step 3: Curve Fitting for Mitoflash Detection
+
+> <hands-on-title>Mitoflash Detection</hands-on-title>
+>
+> 1. {% tool [Perform curve fitting](toolshed.g2.bx.psu.edu/repos/imgteam/curve_fitting/ip_curve_fitting/0.0.3-2) %} with the following parameters:
+>    - {% icon param-file %} *"File name of input data points (xlsx)"*: Select `spots_linked.xlsx`, the output file from the **Perform linking in time series (nearest neighbors)** tool.
+>    - **Degree of the polynomial function**: Set to `2nd degree` to capture intensity peaks that characterize mitoflash events.
+>    - **Penalty**: Choose *Least absolute deviations (LAD)* for robust fitting to intensity fluctuations.
+>    - **Alpha**: Set a significance level, such as `0.01`, to generate assistive curves if needed.
+>
+>    This step applies polynomial curve fitting to model intensity fluctuations over time that represent mitoflash events. After running the tool, key parameters to analyze from the output file include:
```

Comment:
It would be very helpful if you could include the table produced by the tool for the example data you used for this tutorial here, before describing the columns. Like maybe a screenshot. Or maybe also just a section of the table.

Then you could also say how to read the table, for the specific example.